src/components/ProposalHero/ProposalHero.tsx

```diff
@@ -107,6 +114,83 @@ const VoteCTA: React.FC<VoteCTAProps> = ({
   )
 }
 
+type ExecutionDelayCTAProps = {
+  votingDeadlineBlock: number
+}
+
+const ExecutionDelayCTA: React.FC<ExecutionDelayCTAProps> = ({
```

Comment:
added different components for each substate, but could possibly combine the execution delay + awaiting execution into a single component with a flag

Reply:
eh, yeah i think a prop would be nicer, but this is totally fine too